crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -204,6 +214,8 @@ def restore_chef
           raise "Restore failed"
         end
 
+        # allow enable dns after restore.
+        disable_dns_path.delete if disable_dns_path.exist?
```

Comment:
@mmnelemane I was also wondering if we could put this into the cleanup method, or do you think that has to be deleted before restoring the database?